src/test/scala/mesosphere/marathon/api/v2/ModelValidationTest.scala

```diff
@@ -103,7 +103,7 @@ class ModelValidationTest
       case f: Failure =>
         val errors = (Json.toJson(f) \ "details").as[Seq[JsObject]]
         errors should have size 1
-        (errors.head \ "path").as[String] should be("/groups(0)/apps(1)")
+        (errors.head \ "path").as[String] should be("/groups(group1)/apps(invalid)")
```

Comment:
I still wonder if the syntax is not confusing: `/groups(/nested)/apps(/nested/app1)/externalVolumes(0)`